_posts/2024-03-09-neue-server-am-standort-muc.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Neue Server am Standort München seit 09.03.2024"
+date:   2024-03-09 20:00:00
+categories: freifunkmuc
+---
+
+**Neue Server am Standort München seit 09.03.2024**
+
+Am 09. März 2024 wurden an unserem Standort in Münchnen von unserm Team die alten Server gegen neue ausgetauscht.  
+Es stehen jetzt statt der insgesamt 32 Kerne in sehr alten Intels 256 Kerne mit Epycs zur Verfügung:  
```

Comment:
den Satz verstehe ich nicht ganz "in sehr alten Intels 256 Kerne mit Epycs zur Verfügung:"